src/components/screens/PointOfInterest.js

```diff
@@ -76,10 +76,11 @@ export const PointOfInterest = ({ data, hideMap, navigation, route }) => {
   const voucherListItems = useMemo(() => {
     return parseListItemsFromQuery(
       QUERY_TYPES.VOUCHERS,
-      { [QUERY_TYPES.GENERIC_ITEMS]: vouchers },
+      { [QUERY_TYPES.VOUCHERS]: vouchers },
       undefined,
       {
-        withDate: false
+        withDate: false,
+        queryKey: QUERY_TYPES.VOUCHERS
```

Comment:
this should be `QUERY_TYPES.GENERIC_ITEMS` i guess, compared to `query == QUERY_TYPES.VOUCHERS_REDEEMED ? QUERY_TYPES.VOUCHERS : QUERY_TYPES.GENERIC_ITEMS`

Reply:
I think `QUERY_TYPES.VOUCHERS` is correct. Because this is not a `genericItems` query. The data is in the `vouchers` object inside the POI query.  If we update `queryKey` to `genericItems`, it will be `undefined`. Because there is no data inside the `genericItems` object.

Reply:
ah! inside POI object, yes, true, thanks!